docs/agent-k8s-install-helm.md

```diff
@@ -0,0 +1,117 @@
+# Install Using Helm
+
+Use the Helm package manager to install the Smart Agent to Kubernetes
+environments.
+
+If you don't have Helm installed already, use kubectl
+to install the Smart Agent. To learn more, see
+[Install the Smart Agent using kubectl](agent-k8s-install-kubectl.md).
+
+## Prerequisites
+
+* Linux kernel version 2.6.32 or higher
+* cap_dac_read_search and cap_sys_ptrace capabilities
+* Terminal or a similar command-line interface application
+* Helm:
+
+  - The installation chart for the Smart Agent is compatible with Helm version 2 or Helm version 3.
+  - To learn more, see the [Helm site](https://helm.sh/).
+
+* Tiller installed on all of your Kubernetes hosts. Because of
+  the way that the installation chart works, you need Tiller even if you're using
+  Helm version 3.
+* SignalFx realm for your organization. See [Realms](../../../_sidebars-and-includes/smart-agent-realm-note.html).
+* A SignalFx access token. See [Smart Agent Access Token](../../../_sidebars-and-includes/smart-agent-access-token.html).
```

Comment:
org access token

Reply:
In the documentation, we describe two types of tokens:

- UI: User API access token, API: session token
- UI: Access token, API: org token

Since this entry is in the user documentation, I chose to use access token. I can _add_ org token, but I don't think we should use it as a replacement.